### PR TITLE
Use commit hash for `store create` & `app create`

### PR DESCRIPTION
**Pull request by 2can**
## I want to merge this PR because 

- it sets the downloaded repository to specific commit hash

## Related (issues, PRs, topics)

- https://github.com/saleor/saleor-cli/issues/298

## Steps to test feature

- `saleor app create` - commit hash `a5caa3f2580ad075faeee9d4a2125e77bc60f6d8`
- `saleor store create` - commit hash `659ed312a4930e38150276e0ea714efc6ccc22e3`

## I have:

- [x] Tested it locally and it doesn't break existing features
- [ ] Added documentation if public changes are introduced
- [ ] Added tests for my code

### CODE REVIEW

**Comment by zaiste** [comment]
@2can next, let's use some meaningful names for these commits i.e. a tag

**Review by zaiste — APPROVED** [review]

**Comment by lkostrowski** [comment]
Tag is just a pointer to a commit, we can push tags that are CLIs reference, eg `cli-1.16.1` that will help us tracking history

But commits should be enough

File: src/cli/app/create.ts [hunk 1]
```diff
@@ -55,7 +55,11 @@ export const handler = async (argv: Arguments<StoreCreate>): Promise<void> => {
 
   const spinner = ora('Downloading...').start();
   debug('downloading the `master` app template');
-  await downloadFromGitHub('saleor/saleor-app-template', target);
+  await downloadFromGitHub(
+    'saleor/saleor-app-template',
+    target,
+    '659ed312a4930e38150276e0ea714efc6ccc22e3'
```
Comment (lkostrowski) [inline src/cli/app/create.ts:61]: @zaiste @2can please do not keep such stuff inside code. Lets extract it to some root level config file

**Review by lkostrowski — COMMENTED** [review]
Please
1. Extract commit hashes to config file
2. Add option to provide (override) commit hash, eg `saleor app create --commit XYZ`

**Pull request merged**